mk/kind.mk

```diff
@@ -32,7 +26,7 @@ kind/start: ${KUBECONFIG_DIR}
 	$(KIND) get clusters | grep $(KIND_CLUSTER_NAME) >/dev/null 2>&1 && echo "Kind cluster already running." && exit 0 || \
 		($(KIND) create cluster \
 			--name "$(KIND_CLUSTER_NAME)" \
-			--config "$(KUMA_DIR)/test/kind/cluster-$(if $(IPV6),ipv6-,)$(KIND_CLUSTER_NAME).yaml" \
+			--config "$(KUMA_DIR)/test/kind/cluster$(if $(IPV6),-ipv6,).yaml" \
```

Comment:
**Question**: What about the other two sets of files (i.e. the ones with `-kuma-1` and `-kuma-2` suffix)?

Reply:
Should this just be `$(KIND_CONFIG)` and the lines above kept and test/kind-cluster.yaml become test/kind-cluster-kuma.yaml ?

Reply:
I made this change, but I needed to pick between `cluster-ipv6-kuma-1.yaml` and `cluster-ipv6-kuma-2.yaml`. Arbitrarily picked `cluster-ipv6-kuma-1.yaml`. What should we use here?